index.html

```diff
@@ -0,0 +1,240 @@
+            <li><h3>CANOPY</h3></li>
+            <li><img src="../Portfolio-finish-mobile-version/img/circle.svg" alt=""></li>
+            <li><p>Back End Dev</p></li>
+            <li><img src="../Portfolio-finish-mobile-version/img/circle.svg" alt=""></li>
+            <li><p>2015</p></li>
+          </ul>
+          </div>
+          <p>
+            A daily selection of privately
+            personalized reads; no accounts
+            or sign-ups required.
+          </p>
+          <ul class="languages">
+            <li><a href="">html</a></li>
+            <li><a href="">css</a></li>
+            <li><a href="">javascript</a></li>
+            <li><a href="">Ruby on rails</a></li>
+          </ul>
+          <div class="rev">
+            <button><a href="" class="btn">See Project</a></button>
+          </div>
+        </div>
+      </li>
+    </ul>
+    </main>
+  </section>
+  <aside class="about">
+    <div class="abt_head">
+    <h1>About <br>
+        Myself</h1>
+    <p>Hello I’m a software developer!
+        I can help you build a product ,
+        feature or website Look through
+        some of my work and experience!
+        If you like what you see and have
+        a project you need coded, 
+        don’t hestiate to contact me.</p>
+      <h3>
+        LET’S CONNECT
+      </h3>  
+      <ul class="social-media">
+        <li><a href="twiter"><img src="../Project-1/img/twiter.svg" alt=""></a></li>
+        <li><a href="linkedin"><img src="../Project-1/img/Linkedin.svg" alt=""></a></li>
+        <li><a href="medium"><img src="../Project-1/img/medium.svg" alt=""></a></li>
+        <li><a href="github"><img src="../Project-1/img/github.svg" alt=""></a></li>
+        <li><a href="chill"><img src="../Project-1/img/chill.svg" alt=""></a></li>
+      </ul> 
+      <button  class="btn"><a href="">Get my resume</a></button>
+    </div>
+    <div class="links2">
+      <div class="itm1">
+        <div class="lan">
+          <h2>Languages</h2>
+          <img src="./imgoverlay/Union.svg" alt="">
+        </div>
+        
+        <div>
+          <ul>
+            <li><img src="../Portfolio-finish-mobile-version/imgoverlay/Ellipse 1.svg" alt=""></li>
+            <li>javascript</li>
+          </ul>
+          <ul>
+            <li><img src="../Portfolio-finish-mobile-version/imgoverlay/html.png" alt=""></li>
+            <li>Html</li>
+          </ul>
+          <ul>
+            <li><img src="../Portfolio-finish-mobile-version/imgoverlay/css.svg" alt=""></li>
+            <li>Css</li>
+          </ul>
+        </div>
+
+      </div>
+      <div class="itm2">
+        <div class="lan">
+          <h2>Framework</h2>
+          <img src="./imgoverlay/Unionfow.svg" alt="">
+        </div>
+
+      </div>
+      <div class="itm3">
+        <div class="lan">
+          <h2>skilss</h2>
+          <img src="./imgoverlay/Unionfow.svg" alt="">
+        </div>
+
+      </div>
+    </div>   
+  
+  </aside>
```

Comment:
- Hey, please, let's make the aside a section tag to meet the project requirement as this is another section of your webpage to enforce the correct use of the html5 semantic tag